.pipelines/azure-pipelines.yml

```diff
@@ -24,8 +24,8 @@ variables:
   # This is used in functional tests to make certain tests mandatory
   # in CI, but optional during local development. 
   SCITT_CI: 1
-
   ACR_REGISTRY: scittoss.azurecr.io
+  ACR_REGISTRY_RELEASE: confidentialledgeracrprod.azurecr.io
```

Comment:
Do you think it could make sense to push release images to both staging and prod ACRs, so that we can test new releases in staging too? If we only have main-branch images, we cannot use the images specifically created for new SCITT releases

Reply:
yup makes sense, I just made the change and created a new yml template to reduce duplication